framework/include/actions/DetermineSystemType.h

```diff
@@ -28,6 +28,9 @@ class DetermineSystemType : public MooseObjectAction
   DetermineSystemType(InputParameters parameters);
 
   virtual void act() override;
+
+private:
+  InputParameters _action_params;
```

Comment:
`_pars` is for `DetermineSystemType`,  and `_action_params` for `CreateProblemAction`